packages/yoastseo/src/morphology/dutch/addNounSuffixes.js

```diff
@@ -71,24 +87,24 @@ export function addNounSuffixes( stemmedWord, morphologyDataAddSuffixes, morphol
 	const pluralSuffixes = getPluralSuffixes( stemmedWord, morphologyDataNounSuffixes.pluralSuffixes );
 
 	// Get diminutive suffixes.
-	const diminutiveSuffixes = getSuffixes( stemmedWord, morphologyDataNounSuffixes.diminutiveSuffixes );
+	const diminutiveSuffixes = getDiminutiveSuffixes( stemmedWord, morphologyDataNounSuffixes.diminutiveSuffixes );
 
 	// Join the diminutive and plural suffixes
 	let combinedSuffixes = pluralSuffixes.concat( diminutiveSuffixes );
 
 	const nounForms = [];
 
 	/* If the kje suffix exists in the suffixes for this word,
-	remove the last character of the stem, attach the suffix to the stem, and add the resulting
-	form to the noun forms array. */
+	remove the last character of the stem, add the suffixes -kje and -kjes to the stem, and add the resulting
+	forms to the noun forms array. */
 	if ( combinedSuffixes.includes( "kje" ) ) {
-		combinedSuffixes = combinedSuffixes.filter( suffix => suffix !== "kje" );
-		const kjeForm = stemmedWord.slice( 0, -1 ) + "kje";
-		nounForms.push( kjeForm );
+		const kjeSuffixes = combinedSuffixes.filter( suffix => suffix.includes( "kje" ) );
+		combinedSuffixes = combinedSuffixes.filter( suffix => ! suffix.includes( "kje" ) );
+		nounForms.push.apply( nounForms, applySuffixesToStem( stemmedWord.slice( 0, -1 ), kjeSuffixes ) );
```

Comment:
```suggestion
		nounForms.push( ...applySuffixesToStem( stemmedWord.slice( 0, -1 ), kjeSuffixes ) );
```

You can use the spread operator here to make the statement a bit more concise  🙂